server/grpc/gogoreflection/fix_registration.go

```diff
@@ -85,7 +86,7 @@ func getFileDescriptor(filePath string) []byte {
 	if len(fd) != 0 {
 		return fd
 	}
-
+	// nolint: staticcheck
```

Comment:
This is weird: in #9779 (the backport PR) I removed these `nolint`s because the linter said they were [unused](https://github.com/cosmos/cosmos-sdk/runs/3169775342#step:4:77). Now two PRs later, linter is shouting again...

It's not the 1st first we see linter errors on untouched files in PRs, I remember some of these happening on `release/0.42.x` too

Reply:
OK, so:
- if I don't put `nolint`, the error is: "proto.FileDescriptor is deprecated..."
- if I put `nolint`, the error is " directive `// nolint: staticcheck` is unused for linter "staticcheck""

Reply:
I reverted all unrelated files changed in #9779 to their master version. If lint doesn't pass, let's just use admin rights for now to force merge.